WORKSPACE.bazel

```diff
@@ -981,3 +981,9 @@ http_file(
     sha256 = "2a428e8d35518ce22002e498f7d618a9eeeddf57371f965e92cf480dd3cbd046",
     url = "https://github.com/dfinity/examples/releases/download/rust-basic-bitcoin-24-09-16/basic_bitcoin.wasm.gz",
 )
+
+http_file(
+    name = "pocket-ic-mainnet-gz",
+    sha256 = "454891cac2421f3f894759ec5e6b6e48fbb544d79197bc29b88d34b93d78a4f1",
+    url = "https://download.dfinity.systems/ic/52ebccfba8855e23dcad9657a8d6e6be01df71f9/binaries/x86_64-linux/pocket-ic.gz",
```

Comment:
I see mainnet is using `c66489953ed34670667ad81ce42a172948f289f5`.

Will you address syncing this with [testnet/mainnet_revisions.json#L3](https://github.com/dfinity/ic/blob/master/testnet/mainnet_revisions.json#L3) in another PR?

Reply:
> Will you address syncing this with [testnet/mainnet_revisions.json#L3](https://github.com/dfinity/ic/blob/master/testnet/mainnet_revisions.json#L3) in another PR?

Yep, talked to DRE team and we'll be taking over the update of that file too. Let's sync offline